filebeat/input/syslog/config.go

```diff
@@ -0,0 +1,60 @@
+package syslog
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/dustin/go-humanize"
+
+	"github.com/elastic/beats/filebeat/harvester"
+	"github.com/elastic/beats/filebeat/inputsource"
+	"github.com/elastic/beats/filebeat/inputsource/tcp"
+	"github.com/elastic/beats/filebeat/inputsource/udp"
+	"github.com/elastic/beats/libbeat/common"
+)
+
+type config struct {
+	harvester.ForwarderConfig `config:",inline"`
+	Protocol                  common.ConfigNamespace `config:"protocol"`
+}
+
+var defaultConfig = config{
+	ForwarderConfig: harvester.ForwarderConfig{
+		Type: "syslog",
+	},
+}
+
+var defaultTCP = tcp.Config{
+	LineDelimiter:  "\n",
+	Timeout:        time.Minute * 5,
+	MaxMessageSize: 20 * humanize.MiByte,
+}
+
+var defaultUDP = udp.Config{
+	MaxMessageSize: 10240,
```

Comment:
Please humanize this option, as it's done in `defaultTCP`.

Reply:
It would be nice to make bytes a first class citizen in ucfg.

Reply:
@andrewkroh I prefer to have it in the beats repository, this way we can more control `deprecation` if needed.